modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts_{context}"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP Workload Identity cluster, you must generate the cloud credentials manually by using the `ccoctl` binary, and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+* You have configured an {product-title} cluster with GCP Workload Identity by using the Cloud Credential Operator in a manual mode.
+
+.Procedure
+
+. Create a directory to store a `CredentialsRequest` resource YAML file by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file under the `credentials-request` directory, such as, `sample-credential-request.yaml` by applying the following yaml:
```

Comment:
This step can be clarified; suggest:

```
. In the `credentials-request` directory, create a YAML file that contains the following `CredentialsRequest` manifest:
```

I don't think the example file name (`sample-credential-request.yaml`) is necessary because it doesn't come up in the rest of the procedure.